### PR TITLE
reviews controller

### PR DESCRIPTION
**Pull request by OmarBerrada**
set before_action find_review for show method only
created the 4 methods: index, show, new, create
created private methods: review_params & find_review

PLEASE MIGRATE YOUR DATABASE FOR SECURITY

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class ReviewsController < ApplicationController
+  before_action :find_review, only: %i[show]
```
Comment (contatc2) [inline app/controllers/reviews_controller.rb]: you can write only: :show

Comment (contatc2) [inline app/controllers/reviews_controller.rb]: before_action :set_job + define set_job in private
def set_job
 @job = Job.find(params[:job_id])
end

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  def show
+  end
+
+  def new
+    @review = Review.new
+  end
+
+  def create
+    @review = Review.new(review_params)
+    @review.job = @review
```
Comment (contatc2) [inline app/controllers/reviews_controller.rb]: @review.job = @job

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      redirect_to job_review_path(@review)
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def review_params
+    params.require(:review).permit(:description, :stars, :job_id)
```
Comment (contatc2) [inline app/controllers/reviews_controller.rb]: i think we don't need :job_id as we set it in the controller

File: app/controllers/reviews_controller.rb (hunk 1 above)
Comment (OmarBerrada) [reply app/controllers/reviews_controller.rb]: didn't know this syntax

File: app/controllers/reviews_controller.rb (hunk 2 above)
Comment (OmarBerrada) [reply app/controllers/reviews_controller.rb]: you are right

File: app/controllers/reviews_controller.rb (hunk 3 above)
Comment (OmarBerrada) [reply app/controllers/reviews_controller.rb]: not sure about that ...

**Comment by OmarBerrada** [comment]
please verify changes on the reviews-controller

**Pull request merged**